src/sage/algebras/quatalg/quaternion_algebra.py

```diff
@@ -955,7 +955,9 @@ def maximal_order(self, take_shortcuts=True, order_basis=None):
             Order of Quaternion Algebra (-22, 210) with base ring Rational Field
              with basis (1, i, 1/2*i + 1/2*j, 1/2 + 17/22*i + 1/44*k)
 
-            sage: for d in ( m for m in range(1, 750) if is_squarefree(m) ):        # long time (3s)
+            sage: for d in range(1, 700):       # long time (3s)
```

Comment:
To make the test timing comparable with past versions I would keep the 750, but that's really only a personal opinion, feel free to ignore!

```suggestion
            sage: for d in range(1, 750):       # long time (3s)
```